Policy

spinal84 opened the pull request:
Rename and modify policy files according to API docs.

spinal84 committed:
Rename policy files according to API docs

spinal84 committed:
Modify policy_add.c according to API docs

spinal84 committed:
Modify policy_always_online.c according to API doc

spinal84 committed:
Modify policy_any.c according to API docs

spinal84 committed:
Modify policy_iap_ask.c according to API docs

spinal84 committed:
Modify policy_change.c according to API docs

spinal84 committed:
Modify policy_merge.c according to API docs

spinal84 committed:
Modify policy_nw_disconnect.c according to API doc

spinal84 committed:
Modify policy_one.c according to API docs

spinal84 committed:
Modify policy_iap_restart.c according to API docs

freemangordon commented on policy/policy_always_online.c:
```diff
@@ -296,136 +316,205 @@ network_type_notify_cb(GConfClient *client, guint cnxn_id, GConfEntry *entry,
-    data->check_connections_timeout_id =
-        g_timeout_add(500, check_connections_cb, data);
+    data->count_iaps_id =
+        g_timeout_add(POLICY_ALWAYS_ONLINE_IAP_TIMEOUT, policy_always_online_check, data);
```
more than 80 symbols per line, breaks coding style

freemangordon commented on policy/policy_always_online.c:
```diff
@@ -296,136 +316,205 @@ network_type_notify_cb(GConfClient *client, guint cnxn_id, GConfEntry *entry,
-    data->check_connections_timeout_id =
-        g_timeout_add(500, check_connections_cb, data);
+    data->count_iaps_id =
+        g_timeout_add(POLICY_ALWAYS_ONLINE_IAP_TIMEOUT, policy_always_online_check, data);
```
ditto

freemangordon commented on policy/policy_always_online.c:
```diff
@@ -296,136 +316,205 @@ network_type_notify_cb(GConfClient *client, guint cnxn_id, GConfEntry *entry,
     icd_dbus_disconnect_system_bcast_signal(MCE_SIGNAL_IF,
-                                            device_mode_ind_filter, data,
-                                            "member='" MCE_DEVICE_MODE_SIG "'");
+                                            policy_always_online_flightmode_sig, data,
```
ditto. multi-line if() is missing curly braces

freemangordon commented on policy/policy_any.c:
```diff
@@ -183,10 +185,9 @@ scan_cb(const guint status, const gchar *service_name,
-                                      (GCompareFunc)network_compare);
-            scan_data->data->iaps_found = TRUE;
+            data->found_networks =
+                g_slist_insert_sorted(data->found_networks, network, policy_any_sort_network);
```
more than 80 characters

freemangordon commented on policy/policy_any.c:
```diff
@@ -278,229 +279,248 @@ get_max_priority(const GSList *existing_requests)
     {
-      struct scan_data *scan_data;
-
+      struct policy_scan_data *scan_data = policy_any_scan_data_new(
```
not really a good idea to mix renaming to code changes, it becomes very hard to review

freemangordon commented on policy/policy_iap_ask.c:
```diff
@@ -123,20 +125,20 @@ show_conn_dlg_cb(DBusPendingCall *pending, void *user_data)
-                           const GSList *existing_requests,
-                           icd_policy_request_new_cb_fn policy_done_cb,
-                           gpointer policy_token, gpointer *private)
+policy_iap_ask_request(struct icd_policy_request *new_request,
```
coding style?

freemangordon commented on policy/policy_change.c:
```diff
@@ -8,134 +8,161 @@
-  icd_dbus_disconnect_system_bcast_signal(ICD_UI_DBUS_INTERFACE, change_cb,
-                                          data, ICD_UI_FILTER_CHANGE_SIG);
-  cancel_pending_call(data);
+  icd_dbus_disconnect_system_bcast_signal(ICD_UI_DBUS_INTERFACE, policy_change_done,
```
80 chars

freemangordon commented on policy/policy_change.c:
```diff
@@ -8,134 +8,161 @@
 }
 
+static gboolean
+policy_change_confirm(struct policy_change_data *data)
```
merging renaming with code changes

freemangordon commented on policy/policy_change.c:
```diff
@@ -233,9 +238,9 @@ icd_policy_init(struct icd_policy_api *policy_api,
-  icd_dbus_connect_system_bcast_signal(ICD_UI_DBUS_INTERFACE, change_cb, data,
-                                       ICD_UI_FILTER_CHANGE_SIG);
+  policy_api->destruct = policy_change_destruct;
+  icd_dbus_connect_system_bcast_signal(ICD_UI_DBUS_INTERFACE, policy_change_done, data,
```
80 chars per line

freemangordon commented:
not finished with the review yet, don't make a new pull request :)

spinal84 commented:
Updated. No lines > 80 chars now.  
Renaming of files is just the first commit. So I added modifications according to API docs not to make more pull requests dedicated to the policy changes.

spinal84 commented:
About mixing internal names renaming with code changes. You are right.  
I tried to change the sources to make them like described in the API docs.
Should I redo the work to make separate commits for renaming of everything?

freemangordon commented:
yes, please, do at least 2 commits per file - one for renaming of the functions, one for renaming of the structure and members and one more if there are code changes - like moving block of code in a new function. Sorry about that, but it is really hard to review what's going on if all is merged in a single commit per file.

spinal84 commented:
Please check pull request #8
